image/rhel/konflux.Dockerfile

```diff
@@ -65,18 +66,18 @@ WORKDIR /go/src/github.com/stackrox/rox/app
 
 COPY --chown=default . .
 
-# This installs yarn from Cachi2 and makes `yarn` executable available.
-# Not using `npm install --global` because it won't get us `yarn` globally.
-RUN cd .konflux/bootstrap-yarn && \
-    npm ci --no-audit --no-fund
-ENV PATH="$PATH:/go/src/github.com/stackrox/rox/app/.konflux/bootstrap-yarn/node_modules/.bin/"
-
 # This sets branding during UI build time. This is to make sure UI is branded as commercial RHACS (not StackRox).
 # ROX_PRODUCT_BRANDING is also set in the resulting image so that Central Go code knows its RHACS.
 ENV ROX_PRODUCT_BRANDING="RHACS_BRANDING"
+# Default execution of the `npm ci` command causes postinstall scripts to run and spawn a new child process
+# for each script. When building in konflux for s390x and ppc64le architectures, spawing
+# these child processes causes excessive memory usage and ENOMEM errors, resulting
+# in build failures. Currently the only postinstall scripts that run for the UI dependencies are:
+#   `core-js` prints a banner with links for donations
+#   `cypress` downloads the Cypress binary from the internet
+# In the case of building the `rhacs-main-container`, all of these install scripts can be safely ignored.
+ENV UI_PKG_INSTALL_EXTRA_ARGS="--ignore-scripts"
```

Comment:
I would suggest introducing the same `UI_PKG_INSTALL_EXTRA_ARGS="--ignore-scripts"` in the upstream build around
https://github.com/stackrox/stackrox/blob/f993b1dd3f015e8d85cd51df9b1fe94196fda9ce/.github/workflows/build.yaml#L101-L105
or maybe for the entire `pre-build-ui` so that we get to know earlier if this starts causing problems, e.g. due to another package not doing something mandatory because the scripts are ignored.

Even better if you just set `UI_PKG_INSTALL_EXTRA_ARGS` in `ui/Makefile`. This way we get speedup in CPaaS/OSBS and know that our builds are identical w.r.t. this setting.

Reply:
I added this to the entire `pre-build-ui` section. I didn't want to add this directly to `ui/Makefile` since there are some make targets that call `make deps` that _do_ need the postinstall scripts to run (namely anything that eventually runs Cypress). This applies to local Cypress runs as well.